cfy_manager/components/restservice/scripts/create_amqp_resources.py

```diff
@@ -50,4 +50,4 @@ def _get_default_tenant():
     default_tenant = _get_default_tenant()
     amqp_manager.create_tenant_vhost_and_user(default_tenant)
     amqp_manager.sync_metadata()
-    print 'Finished creating AMQP resources'
+    print('Finished creating AMQP resources')
```

Comment:
feel free to just remove this line instead
otherwise do add the future import for print

Reply:
Removed